### PR TITLE
feat: configure tenderdash log level

### PR DESCRIPTION
**Pull request by shumkov**
## Issue being fixed or feature implemented

We need to be able to control tenderdash log output using dashmate configs

## What was done?

Added `platform.drive.tenderdash.log.format` and `platform.drive.tenderdash.log.level` to schema with rules corresponding to tendermint documentation.

## How Has This Been Tested?

Tested locally

## Breaking Changes

Should be covered by migration

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added or updated relevant unit/integration/functional/e2e tests
- [x] I have made corresponding changes to the documentation

**For repository code-owners and collaborators only**
- [x] I have assigned this pull request to a milestone

### CODE REVIEW

File: configs/schema/configJsonSchema.js [hunk 1]
```diff
@@ -359,6 +363,54 @@ module.exports = {
...
+                          $ref: '#/definitions/tenderdashLogModule',
+                        },
+                        statesync: {
+                          $ref: '#/definitions/tenderdashLogModule',
+                        },
+                        '*': {
+                          $ref: '#/definitions/tenderdashLogModule',
+                        },
+                      },
+                      required: ['*'],
```
Comment (shumkov) [inline configs/schema/configJsonSchema.js:403]: Does the wildcard is required? Or `log_level` can't be empty string?

File: configs/schema/configJsonSchema.js [hunk 2]
```diff
@@ -72,6 +72,10 @@ module.exports = {
       additionalProperties: false,
       required: ['level', 'path'],
     },
+    tenderdashLogModule: {
+      type: 'string',
+      enum: ['debug', 'warn', 'info', 'error'],
```
[the remark this answers is not in the document]
Comment (shumkov) [reply configs/schema/configJsonSchema.js:77]: No. It was my guess. Let's remove it then ))

File: configs/schema/configJsonSchema.js (hunk 1 above)
Comment (strophy) [reply configs/schema/configJsonSchema.js:403]: Yes, some minimum `log_level` must be defined, but it doesn't have to be `'*'`, we could set `main` instead. If it is set blank then the Tenderdash container restarts with error `ERROR: empty log level`.

Comment (strophy) [reply configs/schema/configJsonSchema.js:403]: Possible log levels are listed here: https://docs.tendermint.com/master/nodes/logging.html#list-of-modules
I'm not sure what `main` is actually, it is not in the list but is accepted by Tenderdash.

Comment (shumkov) [reply configs/schema/configJsonSchema.js:403]: I see. Then you can use just `minProperties: 1` instead of `required`

**Review by strophy — APPROVED** [review]
LGTM :+1:

**Pull request merged**